### PR TITLE
HttpSender errors are not logged #406

### PR DESCRIPTION
**Pull request by chandresh-pancholi**
Signed-off-by: chandresh-pancholi<chandreshpancholi007@gmail.com>

## Which problem is this PR solving?
- #406 

## Short description of the changes
- Logging errors when the response is not successful and not able to read response body event though it is not null

### CODE REVIEW

File: jaeger-thrift/src/main/java/io/jaegertracing/thrift/senders/HttpSender.java [hunk 1]
```diff
@@ -73,7 +75,9 @@ public void send(Process process, List<Span> spans) throws SenderException {
       try {
         responseBody = response.body() != null ? response.body().string() : "null";
       } catch (IOException e) {
-        responseBody = "unable to read response";
+          responseBody = String.format("Unable to read response: %s", e.getMessage());
```
Comment (jpkrohling) [inline jaeger-thrift/src/main/java/io/jaegertracing/thrift/senders/HttpSender.java]: I should have caught that on the previous PR, but the indentation here seems odd. Compare with the position of `responseBody` in the `try` part of this try-catch.

Other than that, LGTM.

**Comment by jpkrohling** [comment]
Tip: if you add the word "Resolves" (or "Closes") before the item number on the description, the referred issue will be automatically closed when this gets merged.

File: jaeger-thrift/src/main/java/io/jaegertracing/thrift/senders/HttpSender.java (hunk 1 above)
Comment (chandresh-pancholi) [reply jaeger-thrift/src/main/java/io/jaegertracing/thrift/senders/HttpSender.java]: I am using 4 tab indentation. I will change it to 2 tab

**Comment by black-adder** [comment]
The changes look great, can you update some tests to make sure the exception is being logged?

File: jaeger-thrift/src/main/java/io/jaegertracing/thrift/senders/HttpSender.java [hunk 2]
```diff
@@ -21,6 +21,7 @@
 import java.io.IOException;
 import java.util.List;
 import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
```
Comment (jpkrohling) [inline jaeger-thrift/src/main/java/io/jaegertracing/thrift/senders/HttpSender.java]: Looks like this import has been added by mistake?

**Comment by pschichtel** [comment]
I'd say this PR has also been obsoleted by my recently merged #729

**Pull request closed without merging**